website/docs/r/compute_project_metadata.html.markdown

```diff
@@ -8,7 +8,10 @@ description: |-
 
 # google\_compute\_project\_metadata
 
-Manages metadata common to all instances for a project in GCE.
+Manages metadata common to all instances for a project in GCE. If you
+want to manage only single key/value pairs within the project metadata
+rather than the entire map, then use
```

Comment:
Is it being a map an implementation detail? We describe it as a series below in the `metadata` field docs, for example.